packages/core/contracts/optimistic-asserter/implementation/sovereign-security/WhitelistProposerSovereignSecurity.sol

```diff
@@ -6,7 +6,7 @@ import "../../interfaces/OptimisticAsserterInterface.sol";
 
 contract WhitelistProposerSovereignSecurity is BaseSovereignSecurity, Ownable {
```

Comment:
As we are changing `proposer` to `asserter` we could also reflect it in this contract, e.g. `WhitelistAsserterSovereignSecurity` and `whitelistedAsserters`.

Reply:
This also applies to `WhitelistProposerSovereignSecurity.t.sol`